src/Generator/GeneratorInterface.php

```diff
@@ -40,5 +40,5 @@ public function stripDeclaration($xml);
 
     public function flush();
 
-    public function __toString();
+    public function __toString(): string;
```

Comment:
Had to check these again: apparently adding `:string` (return type) is not a BC break on `__toString()`, thanks to the implicit `Stringable` implementation.

In fact, following is legal PHP 8.1 code:

```php
<?php

class Foo {
    function __toString(): string {}
}

class Bar extends Foo {
    function __toString() {}
}
```

Reply:
It was also required for Psalm to pass